chpl/chpl-service/src/main/java/gov/healthit/chpl/changerequest/domain/service/ChangeRequestAttestationService.java

```diff
@@ -17,109 +11,79 @@
 import gov.healthit.chpl.dao.DeveloperDAO;
 import gov.healthit.chpl.dao.UserDeveloperMapDAO;
 import gov.healthit.chpl.dao.auth.UserDAO;
 import gov.healthit.chpl.domain.Developer;
 import gov.healthit.chpl.domain.activity.ActivityConcept;
 import gov.healthit.chpl.dto.auth.UserDTO;
-import gov.healthit.chpl.email.ChplEmailFactory;
-import gov.healthit.chpl.email.ChplHtmlEmailBuilder;
 import gov.healthit.chpl.exception.EmailNotSentException;
 import gov.healthit.chpl.exception.EntityCreationException;
 import gov.healthit.chpl.exception.EntityRetrievalException;
 import gov.healthit.chpl.exception.InvalidArgumentsException;
 import gov.healthit.chpl.exception.UserRetrievalException;
+import gov.healthit.chpl.form.Form;
+import gov.healthit.chpl.form.FormItem;
+import gov.healthit.chpl.form.FormService;
+import gov.healthit.chpl.form.SectionHeading;
+import gov.healthit.chpl.form.validation.FormValidator;
 import gov.healthit.chpl.manager.ActivityManager;
 import gov.healthit.chpl.sharedstore.listing.ListingStoreRemove;
 import gov.healthit.chpl.sharedstore.listing.RemoveBy;
 import gov.healthit.chpl.util.AuthUtil;
-import gov.healthit.chpl.util.DateUtil;
 import lombok.extern.log4j.Log4j2;
 
 @Component
 @Log4j2
 public class ChangeRequestAttestationService extends ChangeRequestDetailsService<ChangeRequestAttestationSubmission> {
     private ChangeRequestDAO crDAO;
-    private ChangeRequestAttestationDAO crAttesttionDAO;
-    private ChplEmailFactory chplEmailFactory;
+    private ChangeRequestAttestationDAO crAttestationDAO;
     private AttestationManager attestationManager;
     private AttestationPeriodService attestationPeriodService;
-    private ChplHtmlEmailBuilder chplHtmlEmailBuilder;
     private UserDAO userDAO;
-    private DeveloperDAO developerDao;
+    private DeveloperDAO developerDAO;
     private ActivityManager activityManager;
-
-    @Value("${changeRequest.attestation.submitted.subject}")
-    private String submittedEmailSubject;
-
-    @Value("${changeRequest.attestation.submitted.body}")
-    private String submittedEmailBody;
-
-    @Value("${changeRequest.attestation.approval.subject}")
-    private String approvalEmailSubject;
-
-    @Value("${changeRequest.attestation.approval.body}")
-    private String approvalEmailBody;
-
-    @Value("${changeRequest.attestation.rejected.subject}")
-    private String rejectedEmailSubject;
-
-    @Value("${changeRequest.attestation.rejected.body}")
-    private String rejectedEmailBody;
-
-    @Value("${changeRequest.attestation.pendingDeveloperAction.subject}")
-    private String pendingDeveloperActionEmailSubject;
-
-    @Value("${changeRequest.attestation.pendingDeveloperAction.body}")
-    private String pendingDeveloperActionEmailBody;
-
-    @Value("${changeRequest.attestation.updated.subject}")
-    private String updatedEmailSubject;
-
-    @Value("${changeRequest.attestation.updated.body}")
```

Comment:
A lot of these values are removed in this file. Are they added somewhere else? Or already in use somewhere else? I'm asking because if they're removed here, and no longer used anywhere, we should remove them from the properties files (I think that's where they come from, right?)

Reply:
FYI I see these properties used in various other classes in the PR in the "email" package. I'm approving the PR.